test/e2e/route_advertisements.go

```diff
@@ -1117,12 +1143,34 @@ var _ = ginkgo.DescribeTableSubtree("BGP: isolation between advertised networks"
 				gomega.Expect(err).NotTo(gomega.HaveOccurred())
 				podNetDefault = nil
 			}
-			if svcNetDefault != nil {
-				err = f.ClientSet.CoreV1().Services(svcNetDefault.Namespace).Delete(context.Background(), svcNetDefault.Name, metav1.DeleteOptions{})
+
+			if svcNodePortNetDefault != nil {
```

Comment:
too much duplicate code
please do a loop through all service names and do the delete inside the loop, writing the same delete code 4 times is cue for doing it in loop..

Reply:
Now only 2 svc for default namespace need to be cleanup.  for others service should be able to be removed when e2e namespace are deleted.